src/psyclone/psyir/symbols/symbol.py

```diff
@@ -72,3 +81,6 @@ def name(self):
         :rtype: str
         '''
         return self._name
+
+    def __str__(self):
```

Comment:
So the str of a symbol now returns its name - is that right? Any particular reason?

Reply:
Yes, that is because now we have generic symbols in the symbol table and when doing the symbol table `view()` these are now printed consistently with other symbol sub-classes.